tests/models.py

```diff
@@ -0,0 +1,34 @@
+import django
+from django.db import models
+try:
+    from django.contrib.auth.models import AbstractBaseUser
+except ImportError:
+    # Django 1.4
+    from django.contrib.auth.models import User as AbstractBaseUser
+
+
+class Django14User(AbstractBaseUser):
```

Comment:
From my understanding, you should be able to use https://pytest-django.readthedocs.org/en/latest/helpers.html#django-user-model, correct?